community/sustainability_effort.md

```diff
@@ -0,0 +1,58 @@
+# Argo Workflows Sustainability Effort
+
+Argo Workflows is seeking more community involvement and ultimately more [Reviewers and Approvers](https://github.com/argoproj/argoproj/blob/main/community/membership.md) to help keep it viable. 
+
+We are trying an Incentive System in which contributors pledge a certain number of hours per week (average) to the project and in return have their PRs prioritized for review.
+
+## Where is help needed?
+
+Help is needed for:
+* [reviewing PRs](#reviewing-PRs)
+* [triaging](../docs/CONTRIBUTING.md#triaging-bugs) new incoming bugs by prioritizing them with `P0`, `P1`, `P2`, and `P3` labels
+* responding to questions in [Github Discussions](https://github.com/argoproj/argo-workflows/discussions)
+* responding to questions in [CNCF Slack](https://argoproj.github.io/community/join-slack) in the `#argo-workflows` and `#argo-wf-contributors` channels 
+
+## Commitment
+
+Argo Workflows has roles as described in the [Contributing](../docs/CONTRIBUTING.md#roles) document.
+Any code contributor in a designated role or with an open Membership request can sign up to participate in this Incentive System.
+
+Participants are expected to try to advance in roles.
+There are different expectations depending on the role:
+- Member and non-member: should average a minimum of 6 hours per week of either [authoring PRs](../docs/CONTRIBUTING.md#authoring-prs) or performing any activities listed above in the ["Where is help needed?"](#where-is-help-needed) section
+  - Note that more time is required in this role in order to learn the codebase and processes to make sufficient progress
+- Reviewer and above: should average a minimum of 2 hours per week performing any activities listed above in ["Where is help needed?"](#where-is-help-needed) section 
+  - Highest priority is to ensure that all PRs labeled ["prioritized-review"](#authoring-prs) have an Assignee
```

Comment:
```suggestion
  - Highest priority is to ensure that all PRs labeled `prioritized-review` [have an Assignee]((#reviewing-prs)
```

- Use `prioritized-review` with backticks instead of quotation, for consistency with the rest of the doc and all other label references
- Linked the "have an Assignee" text instead, as I would expect a link on a label to just go to that label, and not a section of the doc
- Linked to "Reviewing PRs" section as that is the one that has instructions on this